.gitignore

```diff
@@ -32,6 +32,7 @@ gradle-app.setting
 
 # IDEs and editors
 /.idea
+/.vscode
```

Comment:
Just a general note: it's a good practice to add things like these to an own global ignore file (https://sebastiandedeyne.com/setting-up-a-global-gitignore-file/).

Reply:
I don't get it, could you give me more hints?